### PR TITLE
The Hangman Game added

### LINKED ISSUE #16: Front-end The Hangman-Game Website
In this game user have to guess word in limited chances.

Attaching website screenshots here:- 
image
image
image
image

### PR DESCRIPTION
**Pull request by vijay-kumar-yadav**
closes #16

## Changes Proposed
Added project to project folder
Updated Index.html with project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -110,18 +110,19 @@ <h4>Third Project</h4>
...
-                            <img src="img/coming_soon.png" class="img-fluid" alt="Profile" />
+                            <img src="./img/projects/The-Hangman-Game.jpg" class="img-fluid" alt="Profile" />
                         </div>
                         <div class="team-info">
                             <h4>Fourth Project</h4>
-                            <span>Sample Name</span>
+                            <span>The Hangman Game</span>
                             <div class="social">
-                                <a href="#" target="_blank">
+                                <a href="https://vijay-kumar-yadav.github.io/The-Hangman-Game/" target="_blank">
```
Comment (TusharKesarwani) [inline index.html:122]: If you are adding the link then add the link to this project

**Review by TusharKesarwani — COMMENTED** [review]
Delete vs code live server file

**Comment by vijay-kumar-yadav** [comment]
> Delete vs code live server file

fixed and applied chnages

**Comment by vijay-kumar-yadav** [comment]
> > Delete vs code live server file
> 
@TusharKesarwani  fixed and applied chnages

**Review by TusharKesarwani — APPROVED** [review]

**Pull request merged**